src/pyobs/vx04.py

```diff
@@ -155,9 +157,10 @@
 translate_sat = {'Suomi-NPP': 'SNPP'}
 
 
-KX = dict ( SNPP_DT_OCEAN = 336,
-            SNPP_DT_LAND  = 335,
-            SNPP_DB_OCEAN  = 334,
+KX = dict ( SNPP_DT_OCEAN = 337,
+            SNPP_DT_LAND  = 336,
+            SNPP_DB_OCEAN  = 335,
+            SNPP_DB_DEEP   = 334,
```

Comment:
Do we need to update the kx values?

Reply:
yes, i added a new viirs retrieval called db_deep (before it was only db_land and db_ocean).  so I had to add a new kx value.  i had updated this in AeroApps as well.

Reply:
ok, i have updated is so VSNPPDBO = 334 again, and VSNPPDBD = 335 now.  this should still be consistent with past experiments

Reply:
Thanks, it will simplify the comparisons :-).